packages/plugin-credentials/test/unit/spec/credentials.js

```diff
@@ -110,6 +110,13 @@ describe(`plugin-credentials`, () => {
         spark.credentials.supertoken = supertoken;
         spark.credentials.userTokens.add(apiToken);
         spark.credentials.userTokens.add(kmsToken);
+
+        spark.credentials.isLoggingOut = false;
```

Comment:
Add "isLoggingOut" as a session property; then, you can default it to `false` and remove this line